techmatters/terraso-mobile-client
# feat: privacy sheet updates

## Description

Replace old ad-hoc privacy info modal implementation with one built on the standard `OverlaySheet` component. Update sheet content to match specification in design documents. Do minor cleanup in related areas.

### Checklist

### Related Issues
#755

### Verification steps

Press the info button in the following areas and verify that the sheet works as expected:
- Create project form
- Create site form
- Project input screen
- Site dashboard screen
